packages/phoenix-client/src/phoenix/client/experiments/__init__.py

```diff
@@ -0,0 +1,761 @@
+from typing import Any, Mapping, Optional, Union
+
+import httpx
+
+from phoenix.client.client import AsyncClient, Client
+from phoenix.client.resources.datasets import Dataset
+from phoenix.client.resources.experiments.types import (
+    ExperimentEvaluators,
+    ExperimentTask,
+    RanExperiment,
+    RateLimitErrors,
+)
+
+DEFAULT_TIMEOUT_IN_SECONDS = 60
+
+
+def run_experiment(
+    *,
+    base_url: Union[str, httpx.URL, None] = None,
+    api_key: Optional[str] = None,
+    headers: Optional[Mapping[str, str]] = None,
+    http_client: Optional[httpx.Client] = None,
```

Comment:
maybe just make the client an optional param?

Reply:
also there might be a benefit to this being lower down in the args just from an order of precedence perspactive.